internal/syshandlers/syshandler_network_org.go

```diff
@@ -37,30 +37,6 @@ func (sh *systemHandlers) handleOrganizationBroadcast(ctx context.Context, msg *
 		return false, nil
 	}
 
-	signingIdentity := org.Identity
-	if org.Parent != "" {
-		signingIdentity = org.Parent
-		parent, err := sh.database.GetOrganizationByIdentity(ctx, org.Parent)
-		if err != nil {
-			return false, err // We only return database errors
-		}
-		if parent == nil {
-			l.Warnf("Unable to process organization broadcast %s - parent identity not found: %s", msg.Header.ID, org.Parent)
-			return false, nil
-		}
-	}
-
-	id, err := sh.identity.Resolve(ctx, signingIdentity)
-	if err != nil {
-		l.Warnf("Unable to process organization broadcast %s - resolve identity failed: %s", msg.Header.ID, err)
-		return false, nil
-	}
-
-	if msg.Header.Author != id.OnChain {
-		l.Warnf("Unable to process organization broadcast %s - incorrect signature. Expected=%s Received=%s", msg.Header.ID, id.OnChain, msg.Header.Author)
-		return false, nil
-	}
-
```

Comment:
@peterbroadhurst I'm not sure if this is right. This block of code was giving me trouble so I had it commented out for most of time while I was working on this PR. My gut tells me it shouldn't be removed though.